pkg/minikube/cruntime/docker.go

```diff
@@ -692,17 +697,19 @@ func dockerConfigureNetworkPlugin(r Docker, cr CommandRunner, networkPlugin stri
 	}
 
 	opts := struct {
+		PauseImage     string
 		NetworkPlugin  string
 		ExtraArguments string
 	}{
+		PauseImage:     pauseImage,
 		NetworkPlugin:  networkPlugin,
 		ExtraArguments: args,
 	}
 
 	const CRIDockerServiceConfFile = "/etc/systemd/system/cri-docker.service.d/10-cni.conf"
 	var CRIDockerServiceConfTemplate = template.Must(template.New("criDockerServiceConfTemplate").Parse(`[Service]
 ExecStart=
-ExecStart=/usr/bin/cri-dockerd --container-runtime-endpoint fd:// --network-plugin={{.NetworkPlugin}}{{.ExtraArguments}}`))
+ExecStart=/usr/bin/cri-dockerd --container-runtime-endpoint fd:// --pod-infra-container-image={.PauseImage} --network-plugin={{.NetworkPlugin}}{{.ExtraArguments}}`))
```

Comment:
Rebase re-introduced the typo here -- should be `{{.PauseImage}}` (two curlies on each side)

Reply:
Thanks, forgot to pull remote first.